extensions/default/src/Panels/StudyBrowser/PanelStudyBrowser.tsx

```diff
@@ -8,9 +8,37 @@ import { Separator } from '@ohif/ui-next';
 import { PanelStudyBrowserHeader } from './PanelStudyBrowserHeader';
 import { defaultActionIcons } from './constants';
 import MoreDropdownMenu from '../../Components/MoreDropdownMenu';
+import { CallbackCustomization } from 'platform/core/src/types';
 
 const { sortStudyInstances, formatDate, createStudyBrowserTabs } = utils;
 
+const defaultDoubleClickHandler =
```

Comment:
we put all the customization now in the customizations folder in each extension, register the default there, also update the docs please, i wonder there might be one that does this right now https://docs.ohif.org/platform/services/customization-service/customizationService#customizable-parts-of-ohif